src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -987,7 +987,12 @@ void EIRPlantLoopHeatPump::sizeSrcSideWSHP(EnergyPlusData &state)
         // First the definition of COP: COP = Qload/Power, therefore Power = Qload/COP
         // Then the energy balance:     Qsrc = Qload + Power
         // Substituting for Power:      Qsrc = Qload + Qload/COP, therefore Qsrc = Qload (1 + 1/COP)
-        Real64 const designSourceSideHeatTransfer = tmpCapacity * (1 + 1 / this->referenceCOP);
+        Real64 designSourceSideHeatTransfer = 0.0;
+        if (this->EIRHPType == DataPlant::PlantEquipmentType::HeatPumpEIRHeating) {
+            designSourceSideHeatTransfer = tmpCapacity * (1 - 1 / this->referenceCOP);
+        } else {
+            designSourceSideHeatTransfer = tmpCapacity * (1 + 1 / this->referenceCOP);
+        }
```

Comment:
This should be `tmpCapacity / (1 + 1 / this->referenceCOP)` for heating. I tried both ways and don't get the same answer.

designSourceSideHeatTransfer = Hcap * (1 - 1/COP) = 1000 * (1 - 1/3) = 667
designSourceSideHeatTransfer = Hcap / (1 + 1/COP) = 1000 / (1 + 1/3) = 750

And as a test, if the cooling and heating EIR HP capacity is hardsized and the heating capacity is 1 + 1/COP the size of the cooling capacity (and the COPs are the same), then the autosized load side water flow rate on the cooling EIR HP should be the same size as the autosized source side water flow rate of the heating EIR HP.

Reply:
The above formulations are based on the first-law energy balance equations and COP definition as shown below: 

`Qcond = Qevap + Power; COPc = Qevap / Power; COPh = Qcond / Power`

You can derive them these three equations. The heat transfer should not be the same even if the COP are equal because the heat transfer at the condenser includes the power input while the heat transfer at the evaporator doesn't.

Reply:
The only places that needed to be changed were in source side sizing calculation functions: `sizeSrcSideASHP()` and `sizeSrcSideWSHP()`.

Reply:
@rraustad I searched for `(1 + 1/COP)` and `(1.0 + 1.0 / this->referenceCOP)` but did not find any.

Reply:
Thanks for working through this description, it's helpful.